save-cosv-frontend/src/main/kotlin/com/saveourtool/save/cosv/frontend/components/views/vuln/CosvSchemaView.kt

```diff
@@ -0,0 +1,166 @@
+/**
+ * View, which represents the COSV schema
+ */
+
+package com.saveourtool.save.cosv.frontend.components.views.vuln
```

Comment:
delete vuln directory from the SAVE frontend, that will prevent the confusion and invalid changes

Reply:
Deleted